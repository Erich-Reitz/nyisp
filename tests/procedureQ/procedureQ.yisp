(PROCEDURE? +) ; true

(set a 5) ; nil

(PROCEDURE? a) ; nil

(define fn (x) (+ x 1)) ; fn not for false, but cause its the name

(PROCEDURE? fn) ; true
