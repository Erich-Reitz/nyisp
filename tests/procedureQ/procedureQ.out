t
nil
nil
fn
t
